toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -316,6 +290,16 @@ func (im *IsoMaker) setUpIsoGrub2Bootloader() (err error) {
 		}
 	}
 
+	err = mount.CleanClose()
+	if err != nil {
+		return fmt.Errorf("failed to unmount efiboot.img:\n%w", err)
+	}
+
+	err = loopback.CleanClose()
+	if err != nil {
+		return fmt.Errorf("failed to disconnect efiboot.img:\n%w", err)
+	}
+
```

Comment:
Why not do this in the `defer` calls above?

Reply:
`Close` and `CleanClose` have different meanings. `CleanClose` is for synchronously closing the resource, including ensuring that everything has been successfully flushed to disk. In contrast, `Close` is for making a best effort to ensure the resource is closed, without any regard for data integrity.

For example, `CleanClose` calls unmount without the `MNT_DETACH` flag. Whereas, `Close` calls unmount with the `MNT_DETACH` flag.